public/app/features/dashboard-scene/scene/Scopes/api/dashboards.ts

```diff
@@ -0,0 +1,18 @@
+import { Scope, ScopeDashboardBinding } from '@grafana/data';
+import { getBackendSrv } from '@grafana/runtime';
+
+import { group, namespace, version } from './common';
+
+const endpoint = `/apis/${group}/${version}/namespaces/${namespace}/find/scope_dashboard_bindings`;
+
+export async function fetchDashboards(scopes: Scope[]): Promise<ScopeDashboardBinding[]> {
```

Comment:
I think this belongs to scopes resource client, don't see a reall reason why we would not keep scopes apis together.